### PR TITLE
First Readme update

### PR DESCRIPTION
**Pull request by yildiztugba**
I updated the readme. Please check it out and let me know what else I can add / change.
Later, we can add other parts and information based on our work.

### CODE REVIEW

File: bookfinder/README.md
Comment (EmmanuelOloke) [inline bookfinder/README.md:1]: Hey @yildiztugba Great work on this README. Really awesome detail.

- One major issue with the `Setup` section though is that our app's Clerk Authentication Credentials have been committed and included in this PR.

- For security reasons, it is not advisable to push app credentials to GitHub both in the code base and in the README file.

- You can remove the credentials and replace them with placeholder texts and push again for now.

- Ultimately, I think we'd need to find a way to remove this commit from our git history or just create new credentials altogether. I think the latter would be easier to achieve, but we can discuss more during the meeting later.

Once again, great work with the documentation. I really love it!

**Review by EmmanuelOloke — CHANGES_REQUESTED** [review]

**Review by EmmanuelOloke — DISMISSED** [review]

File: bookfinder/README.md
Comment (yildiztugba) [reply bookfinder/README.md:1]: wow thanks for catching this big issue. I didn't think about that totally. Yes, we need to discuss it later on how to solve it. I fixed it for now.

Comment (EmmanuelOloke) [reply bookfinder/README.md:1]: You're welcome. Happy to help. That's why we have these processes, and we're all learning together.

File: bookfinder/README.md [hunk 1]
```diff
@@ -1,34 +1,88 @@
...
+
+### Frontend
+
+- Tailwind Css
+
+
+
+### Backend
+
+- NextJS
```
Comment (ceifeirocv) [inline bookfinder/README.md:57]: next should be in both back and front end

File: bookfinder/README.md [hunk 2]
```diff
@@ -1,34 +1,88 @@
...
+- React
```
Comment (ceifeirocv) [inline bookfinder/README.md]: react is frontend

File: bookfinder/README.md [hunk 3]
```diff
@@ -1,34 +1,88 @@
...
+- Javascript
```
Comment (ceifeirocv) [inline bookfinder/README.md]: javascrit is both

Comment (yildiztugba) [reply bookfinder/README.md]: done, thanks

**Review by ceifeirocv — APPROVED** [review]

**Review by EmmanuelOloke — APPROVED** [review]

**Pull request merged**